templates/usr/local/bin/haproxy-ocsp-deploy.j2

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Ansible managed
+#
+# set -x;
+set -e;
+set -o pipefail;
+#
+thisFile="$(readlink -f "${0}")";
+thisFilePath="$(dirname "${thisFile}")";
+
+for path in $(ls -1d {{ haproxy_scripts_ssl_src_path }}/*/); do
+  cert="$(basename $path)";
+
+  prefix="100";
+  removePrefix="000";
+
+  if [ $cert == "{{ haproxy_scripts_ssl_first_cert }}" ]; then
+    prefix="000";
+    removePrefix="100";
+  fi
+
+  ocspUrl=$(openssl x509 -noout -ocsp_uri -in ${path}{{ haproxy_scripts_ssl_cert_name }});
+
+  openssl ocsp -no_nonce -respout "{{ haproxy_global_crt_base }}/${prefix}-${cert}.ocsp" \
+    -issuer ${path}{{ haproxy_scripts_ssl_chain_name }} \
+    -verify_other ${path}{{ haproxy_scripts_ssl_chain_name }} \
+    -cert ${path}{{ haproxy_scripts_ssl_cert_name }} \
+    -url ${ocspUrl};
+
+  rm -f "{{ haproxy_global_crt_base }}/${removePrefix}-${cert}.ocsp";
+done
+
+systemctl reload haproxy;
```

Comment:
This should be configurable, or dynamic I guess

Reply:
When don't you want this (after a refresh of the certificates)?